src/bulk/common.ts

```diff
@@ -938,38 +957,39 @@ class BulkOperationBase {
     }
   }
 
+  get forceServerObjectId(): boolean {
+    const result =
+      typeof this.s.options.forceServerObjectId === 'boolean'
+        ? this.s.options.forceServerObjectId
+        : this.s.collection.s.db.options?.forceServerObjectId;
+    return Boolean(result);
+  }
+
   /**
    * Add a single insert document to the bulk operation
    *
-   * @param document the document to insert
-   * @throws {MongoError}
-   * @returns {BulkOperationBase} A reference to self
-   *
    * @example
+   * ```ts
    * const bulkOp = collection.initializeOrderedBulkOp();
    * // Adds three inserts to the bulkOp.
    * bulkOp
    *   .insert({ a: 1 })
    *   .insert({ b: 2 })
    *   .insert({ c: 3 });
    * await bulkOp.execute();
+   * ```
    */
-  insert(document: any): BulkOperationBase {
-    if (this.s.collection.s.db.options.forceServerObjectId !== true && document._id == null)
-      document._id = new ObjectId();
-    return this.s.options.addToOperationsList(this, INSERT, document);
+  insert(document: Document): BulkOperationBase {
+    if (this.forceServerObjectId !== true && document._id == null) document._id = new ObjectId();
+    return this.s.options.addToOperationsList!(this, INSERT, document);
   }
 
   /**
    * Builds a find operation for an update/updateOne/delete/deleteOne/replaceOne.
    * Returns a builder object used to complete the definition of the operation.
    *
-   * @function
-   * @param selector The selector for the bulk operation. See {@link https://docs.mongodb.com/manual/reference/command/update/#update-command-q q documentation}
-   * @throws {MongoError} if a selector is not specified
-   * @returns {FindOperators} A helper object with which the write operation can be defined.
-   *
    * @example
+   * ```ts
```

Comment:
[super nit] I'd keep this as js for our examples, this nit is made even more super by the class being internal 😅  so completely up to you

Reply:
saw you did some comments in `js` guess it's ok unless it's really `ts`